extensions/wikia/AdEngine/js/run/mercury.run.js

```diff
@@ -53,7 +59,13 @@ require([
 	mercuryListener.onLoad(function () {
 		var isRubiconFastlanePrebidAdapterActive = adContext.getContext().bidders.fastlane;
 
-		if (geo.isProperGeo(instantGlobals.wgAmazonMatchCountriesMobile)) {
+		if (geo.isProperGeo(instantGlobals.wgAdDriverA9BidderCountries)) {
+			a9.call();
+		}
+
+		// TODO ADEN-5756 remove 'if' after A9 full roll out
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)